main.nf

```diff
@@ -1032,7 +1132,7 @@ process clean_downloaded_fastq {
     set val(sample_id), val(files_list) from DOWNLOADED_FASTQ_CLEANUP_READY
 
   when:
-    params.output.publish_downloaded_fastq == false
+    params.output.publish_fastq == false
```

Comment:
I think we should leave this name as `publish_downloaded_fastq` to differentiate between local fastq files and dowloaded fastq files.

Reply:
Further... we will never publish local fastq files, as they are already on the local machine. We only need to publish downloaded FASTQ files.